app/assets/javascripts/utilities/buildArticleHTML.js

```diff
@@ -260,7 +270,9 @@ function buildArticleHTML(article) {
       saveButton = `
         <button type="button"
           class="crayons-btn crayons-btn--secondary crayons-btn--icon-left fs-s bookmark-button article-engagement-count engage-button follow-action-button follow-user"
-          data-info='{"id": ${article.id},"className":"User", "name": "${article.user.name}"}'
+          data-info='{"id": ${
```

Comment:
This line breaking appears to be part of the pre-commit hooks. I swear I didn't do this.

Reply:
This is something we should address whenever eslint or the prettier package updates. It can cause formatting issues because the upgrades have tweaked the formatting.